### PR TITLE
fix add_bracket_after_function had no effect

### PR DESCRIPTION
**Pull request by Brow71189**
The type of a completion was compared to "Function" instead of "function" when the add_bracket_after_function flag was checked. This caused add_bracket_after_function to have no effect.

### CODE REVIEW

**Comment by davidhalter** [comment]
Thanks for your contribution!

Please add a test that failed with the old code and I'm happy to merge :)

**Comment by Brow71189** [comment]
I added the test (and confirmed that it actually failed with the old code).
I wasn't exactly sure where to put it, but "test_settings.py" seemed a good place. I'm also happy to change that if you want it somewhere else.

File: test/test_api/test_settings.py [hunk 1]
```diff
@@ -21,3 +21,13 @@ def test_add_dynamic_mods(Script):
     result = script.goto_definitions()
     assert len(result) == 1
     assert result[0].description == 'class int'
+
+
+def test_add_bracket_after_function(Script):
+    api.settings.add_bracket_after_function = True
```
Comment (davidhalter) [inline test/test_api/test_settings.py]: The only problem with this is that you set it globally for all tests that run after it. Can you please use the `monkeypatch` fixture? :) Thanks!

Comment (Brow71189) [reply test/test_api/test_settings.py]: that should do it :)

**Comment by davidhalter** [comment]
Thanks a lot for your contribution :)

**Pull request merged**